code/datums/components/crafting/tailoring.dm

```diff
@@ -402,3 +402,14 @@
 /datum/crafting_recipe/balloon_vest/check_requirements(mob/user, list/collected_requirements)
 	return HAS_TRAIT(user, TRAIT_BALLOON_SUTRA)
 
+/datum/crafting_recipe/voiceconcealer
+	name = "Voice Obfuscator Gas Mask"
```

Comment:
Unironically this sounds on paper stronger than the actual syndicate version since trying to disguise as someone won't show to security that you're not an actual crewmember and they could assume you're just a greyshitter upon seeing you.